services/transaction_record.proto

```diff
@@ -116,4 +116,10 @@ message TransactionRecord {
      * transaction that spawned it.
      */
     Timestamp parent_consensus_timestamp = 15;
+
+    /**
+     * In the record of an internal CryptoCreate transaction triggered by a CryptoTransfer to the alias,
+     * decoded alias of the auto created account
```

Comment:
I wouldn't characterize it as a `decoded alias` since its the raw bytes the client sent. We might also want to make this comment future proof when we add CREAT2 or other use cases.

```suggestion
     * Contains the alias that triggered this account's auto creation.
```